tests/ssh/test_ssh_limit.py

```diff
@@ -66,13 +66,13 @@ def modify_templates(duthost, tacacs_creds, creds):     # noqa F811
     try:
         # Duthost shell not support run command with J2 template in command text.
         admin_session = ssh_connect_remote(
-            dut_ip, creds['sonicadmin_user'], creds['sonicadmin_password'])
+            duthost, dut_ip, creds['sonicadmin_user'], [creds['sonicadmin_password']])
     except paramiko.AuthenticationException:
         # try ssh with ansible_altpassword again
         sonic_admin_alt_password = duthost.host.options['variable_manager']._hostvars[duthost.hostname].get(
             "ansible_altpassword")
         admin_session = ssh_connect_remote(
-            dut_ip, creds['sonicadmin_user'], sonic_admin_alt_password)
+            duthost, dut_ip, creds['sonicadmin_user'], [sonic_admin_alt_password] + creds["ansible_altpasswords"])
```

Comment:
Looks like `ssh_connect_remote` support trying multiple passwords. Why not pass the extra passwords to the `ssh_connect_remote` function in `try` block?

Reply:
Oh, I just kept the original struction, we can combine them here.